testdata/p4_16_samples_outputs/generic-struct-i4835.p4

```diff
@@ -0,0 +1,50 @@
+#include <core.p4>
```

Comment:
Somewhat of a nonstandard issue name for the file? Although we do not really have a consistent naming scheme.

Reply:
oh, sometimes I forget which conventions differ between projects, renamed to be more consistent :-)